data_steward/resources.py

```diff
@@ -125,3 +139,15 @@ def cdm_schemas(include_achilles=False, include_vocabulary=False):
             if include_table:
                 result[table_name] = schema
     return result
+
+
+def hash_dir(in_dir):
+    """
+    Generate an MD5 digest from the contents of a directory
+    """
+    fs = os.listdir(in_dir)
```

Comment:
This is one of the areas some static analysis tools complain about and others do not.  Variable names that are too short are picked up by pylint, but not pycharm.  The idea is that if we are writing self documenting code (few comments), then our variable names should reflect the data they hold.  So, do we want to implement something at a team level that points out areas where we have variable names that are possibly too short?  If so, I would suggest making this segment:

`filelist = os.listdir(in_dir)`
`hash_obj = hashlib.md5()`
`for filename in filelist:`
`    filepath = os.path.join(in_dir, filename)`
`    hash_obj.update(open(filepath, 'rb').read())`
`return hash_obj.hexdigest()`

Reply:
You're right about variable names. I tried to address as many as I could find in this PR. Please feel free to fix any I may have missed. Moving forward I'll make them more descriptive.